.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules/
 bin/node*
-lib/
+/lib/
```

Comment:
Noted in 5477e05

> This is to avoid ignoring the `lib/` directory within packages within `templates/` which requires force adding templates.

So basically to only ignore the top level `lib`.